backend/backend.go

```diff
@@ -206,3 +207,17 @@ func WithVerifierKZGFoldingHashFunction(hFunc hash.Hash) VerifierOption {
 		return nil
 	}
 }
+
+// SupportedCurves returns a list of curves that gnark backends
+// (GROTH16 and PLONK) have been implemented for.
+func SupportedCurves() []ecc.ID {
```

Comment:
This is already available in root package method `Curves()`. Better not to duplicate.